integration/migration_test.go

```diff
@@ -15,34 +15,47 @@
 package integration
 
 import (
-	"github.com/coreos/etcd/pkg/types"
-	"net"
 	"os/exec"
 	"testing"
+	"time"
+
+	"github.com/coreos/etcd/client"
+
```

Comment:
remove the empty line? we have made the decision to treat godeps as in pkg dependencies, right?